app/src/androidTest/java/com/example/android/architecture/blueprints/todoapp/data/source/local/TaskDaoTest.kt

```diff
@@ -18,207 +18,9 @@ package com.example.android.architecture.blueprints.todoapp.data.source.local
 
 import androidx.room.Room
 import androidx.test.core.app.ApplicationProvider.getApplicationContext
-import androidx.test.ext.junit.runners.AndroidJUnit4
-import androidx.test.filters.SmallTest
 import junit.framework.TestCase.assertEquals
-import junit.framework.TestCase.assertNotNull
-import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.first
```

Comment:
Nit: I'm not sure about leaving imports in the empty file. I'd probably remove them as they can give you maintenance headaches in the future if you change the `end` branch.

Reply:
People can look at the code in  the `end` branch in case they get stuck and don't know the right imports, for example

Reply:
Usually I'd agree with you but adding the imports to the starting state removes a **lot** of visual noise and faffing with imports from the codelab instructions. 

Do you think it's a big problem to update the start state as well as the end state if the codelab is update in future?

Reply:
As per chat. Imports removed.